docs/OV_Runtime_UG/migration_ov_2_0/preprocessing.md

```diff
@@ -0,0 +1,68 @@
+# Preprocessing {#openvino_2_0_preprocessing}
+
+## Introduction
+
+Inference Engine API has preprocessing capabilities which are built on top of model expressed as `ngraph::Function` (now, it's `ov::Model`). One of the problems with it - preprocessing operations are not a part of the main model graph, so they are stored and executed separatelly:
+- Preprocessing stored in `InferenceEngine::CNNNetwork` was lost during saving back to IR file format.
+- Preprocessing operations are executed on CPU processors for most of the inference plugins. So, instead of occupying of acceleators, CPU processor is also busy with inference tasks.
+
+OpenVINO Runtime API 2.0 introduces [new way of adding preprocessing operations to the model](../preprocessing_overview.md) - each preprocessing or postprocessing operation is inserted directly to the model and compiled together with inference part of the model via a single ov::Core::compile_model call.
+Having preprocessing operations as a part of official OpenVINO opset allows to read and serialize preprocessed model to the IR file format.
+
+In OpenVINO Runtime 2.0 the following new operations are introduced to be a part of OpenVINO `opset8`:
+- [NV12toRGB](../../ops/image/NV12toRGB_8.md)
+- [NV12toBGR](../../ops/image/NV12toBGR_8.md)
+- [I420toBGR](../../ops/image/I420toBGR_8.md)
+- [I420toRGB](../../ops/image/I420toRGB_8.md)
+All other preprocessing operations are expressed by means of existing operations and their combinations.
+
+It's also important to mention that since OpenVINO 2.0, the Runtime API does not assume any default layouts like Inference Engine did, for example `{ 1, 224, 224, 3 }` is supposed to be `NCHW` layout while it's not true. So, some preprocessing capabilities in OpenVINO Runtime API 2.0 requires explicitly set layouts, see [Layout overview](../layout_overview.md) how to do it. For example, to perform image scaling by partial dimensions `H` and `W`, preprocessing needs to know what dimensions are `H` and `W`.
```

Comment:
`while it's not true` sounds to me like 'everything was bad, now it is good' (but 'bad' is kept for compatibility reasons). Maybe add some link to section below regarding layout conversion?

Reply:
rephrased a bit not to keep `bad` semantic